angular/src/app/index/index.component.html

```diff
@@ -0,0 +1,39 @@
+<header>
+  <img src="./assets/images/header.png" alt="Oupsie, this doesn't work" width="100%" height="100%">
+</header>
+
+<div id="bg">
+  <div class="row valign-wrapper">
+    <div class="col l3 m4 center-align">
+      <img class="responsive-img hide-on-small-only" src="./assets/images/pipidon1.png" alt="Pipidon.png">
+    </div>
+    <div id="pipidon1" class="col l9 m8 s12 card-panel z-depth-2 center-align">
```

Comment:
Same comment as before about using an `id` for CSS selector maching.

And this time you should actually use two `class`es to, as you said on Discord, factor out the common parts.